hooks/src/App.tsx

```diff
@@ -1,32 +1,78 @@
 import './App.css';
-import { EXPIRY_DATE_KEY } from './lib/constants';
+import Input from './Input';
+import InputField from './InputField';
+import { useCardNumbers, useCVC } from './lib';
 import useExpiryDate from './lib/expiryDate/useExpiryDate';
 
 function App() {
-  const { expiryDate, validationResults, handleExpiryDateChange } =
-    useExpiryDate();
+  const cardNumbersResult = useCardNumbers();
+  const expiryDateResult = useExpiryDate();
+  const cvcResult = useCVC();
 
   return (
-    <>
-      <h1>Hooks Modules</h1>
-      <input
-        type="text"
-        name={EXPIRY_DATE_KEY.month}
-        onChange={handleExpiryDateChange}
-        value={expiryDate.month}
-      />
-      <p>{validationResults.month.isValid}</p>
-      <input
-        type="text"
-        name={EXPIRY_DATE_KEY.year}
-        onChange={(e) => handleExpiryDateChange(e, false)}
-        value={expiryDate.year}
-      />
-      <p>{validationResults.year.isValid}</p>
-
-      <p>{validationResults.month.errorMessage}</p>
-      <p>{validationResults.year.errorMessage}</p>
-    </>
+    <section className="section">
+      <div className="sectionContainer">
+        <InputField
+          feedbackMessage={cardNumbersResult.validationResults.errorMessage}
+          title="결제할 카드 번호를 입력해주세요."
+          description="본인 명의의 카드만 결제 가능합니다."
+          label={`
+            카드 번호${
+              cardNumbersResult.network !== ''
+                ? ` (${cardNumbersResult.network})`
+                : ''
+            }`}
+        >
+          <Input
+            type="text"
+            name="cardNumber"
+            value={cardNumbersResult.cardNumberFormatter(
+              cardNumbersResult.cardNumbers
+            )}
+            handleInputChange={(e) =>
+              cardNumbersResult.handleCardNumbersChange(e, false)
+            }
+            isValidInput={cardNumbersResult.validationResults.isValid}
+          />
+        </InputField>
+        <InputField
+          feedbackMessage={
+            expiryDateResult.validationResults.month.errorMessage
+          }
+          title="카드 유효기간을 입력해 주세요."
+          description="월/년도(MMYY)를 순서대로 입력해 주세요."
```

Comment:
소소하게 아래처럼 되어 있으면 좋지 않을까 하는 생각이 드네요 

```suggestion
          description="월/년도(MM/YY)를 순서대로 입력해 주세요."
```

Reply:
오 더 직관적이고 좋은 것 같습니다!

Reply:
카드 유효기간 입력에 대한 설명을 `(MMYY)` 에서 `(MM/YY)`로 수정했습니다!

[refactor: 카드 유효기간 입력에 대한 inputField의 설명 수정 (f1eeb70)](https://github.com/woowacourse/react-modules/pull/157/commits/f1eeb70609f23a55318bbec35fd6255ff91e131b)